src/demo-app/datepicker/datepicker-demo.html

```diff
@@ -39,6 +39,17 @@ <h2>Options</h2>
   </mat-form-field>
 </p>
 
+<h2>Datepicker with custom header</h2>
```

Comment:
Move this section to the bottom of the file

Reply:
done in b6fc003e8919b4a898c4bc8fe0a23f4eecc2e43a